app/views/my_approval_requests/index.html.haml

```diff
@@ -34,7 +34,7 @@
             %td
               = link_to req.requester.name, req.requester
             %td
-              = req.created_at.strftime(ApplicationHelper.get_date_format)
+              = req.created_at.strftime("%B %d, %Y  %r")
```

Comment:
Would have been great if we centralized the date format. That is specify this date format in the function `ApplicationHelper.get_date_format`